packages/headless/src/controllers/core/generated-answer/headless-core-generated-answer.ts

```diff
@@ -275,6 +279,11 @@ export function buildCoreGeneratedAnswer(
     dispatch(registerFieldsToIncludeInCitations(fieldsToIncludeInCitations));
   }
 
+  const expanded = props.initialState?.expanded;
+  if (expanded) {
+    dispatch(expandGeneratedAnswer());
```

Comment:
We can add a small unit tests that confirms that this action has been called when the initial state `expanded` is passed.